src/utils/sort.ts

```diff
@@ -0,0 +1,23 @@
+/** Function that sorts arrays or Float64Arrays/Float32Arrays in ascending order
+ * @param array - array to sort
+ * @returns sorted array
+ */
+export function sortAscending(array: any) {
+  if (array instanceof Float32Array || array instanceof Float64Array) {
+    return array.sort();
+  } else if (array instanceof Array) {
+    return array.sort((a, b) => (a > b ? 1 : -1));
+  }
+}
+
+/** Function that sorts arrays or Float64Arrays/Float32Arrays in descending order
+ * @param array - array to sort
+ * @returns sorted array
+ */
+export function sortDescending(array: any) {
```

Comment:
Should go in 'x' folder and be named xSortDescending

Array should be of type 'DoubleArray'